tutorials/introductory/quilt_layout.py

```diff
@@ -0,0 +1,252 @@
+"""
+==============
+Patchwork Axes
+==============
+
+This is inspired by the `patchwork
+<https://github.com/thomasp85/patchwork>`__ library for R.  While we
+do not implement the operator overloading style, we do provide a
+pythonic API for specifying (nested) Axes layouts.  If you need to do
+something that is not clearly expressed using this method, please see
+:ref:`sphx_glr_tutorials_intermediate_gridspec.py`.
+
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def label_axes(ax_dict):
+    """
+    Helper to set the title on the Axes.
+
+    Parameters
+    ----------
+    ax_dict : Dict[str, Axes]
+        Mapping between the title / label and the Axes.  Set the left
+        title to be equal to the key.
+    """
+    for k, ax in ax_dict.items():
+        ax.set_title(k, loc="left")
+
```

Comment:
Suggest a simple layout first with `subplots(2, 2)` contrasted with `subplot_mosaic("AB\nCD")`, and making it clear that `ax['A']` is a dictionary with the same thing in it as `ax[0, 0]`.